mcs_benchmark_data/pipelines/anli/anli_benchmark_pipeline.py

```diff
@@ -12,7 +12,7 @@
 
 
 class AnliBenchmarkPipeline(_Pipeline):
-    ID = "ANLI"
+    ID = "aNLI"
```

Comment:
Let's stick with Python file naming conventions (snake_case), so that it's `data/anli` and `mcs_benchmark_data/pipelines/anli`. All lower case.